sensing/radar_tracks_noise_filter/test/radar_tracks_noise_filter/test_radar_tracks_noise_filter_isNoise.cpp

```diff
@@ -0,0 +1,70 @@
+// Copyright 2023 Tier IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "radar_tracks_noise_filter/radar_tracks_noise_filter_node.hpp"
+
+#include <radar_msgs/msg/radar_scan.hpp>
+
+#include <gtest/gtest.h>
+
+std::shared_ptr<radar_tracks_noise_filter::RadarTrackCrossingNoiseFilterNode> get_node(
+  float velocity_y_threshold)
+{
+  rclcpp::NodeOptions node_options;
+  node_options.parameter_overrides({
+    {"node_params.is_amplitude_filter", true},
+    {"velocity_y_threshold", velocity_y_threshold},
+  });
+
+  auto node =
+    std::make_shared<radar_tracks_noise_filter::RadarTrackCrossingNoiseFilterNode>(node_options);
+  return node;
+}
+
+radar_msgs::msg::RadarTrack get_radar_track(float velocity_y)
+{
+  radar_msgs::msg::RadarTrack radar_track;
+  radar_track.velocity.y = velocity_y;  // 修正: 変数名を radar_track に修正
+  return radar_track;
+}
```

Comment:
1. [Must] Use camel case like 'getRadarTrack' in ROS2 rule.
2. [IMO] Avoid implicit defaults value (minimum for using variable).
3. [IMO] It is better for test with different variable. In actual case, radar_track.velocity.x has different value other than 0.
4. [IMO] When you initialize ROS2 variable, it is better to use `geometry_msgs::build<geometry_msgs::msg::Vector3>().x(x_).y(y_).z(z_);` grammar for simple coding.

```
radar_msgs::msg::RadarTrack getRadarTrack(float velocity_x, float velocity_y)
{
  radar_msgs::msg::RadarTrack radar_track;
  radar_track.velocity = geometry_msgs::build<geometry_msgs::msg::Vector3>().x(velocity_x).y(velocity_y).z(0.0)
  return radar_track;
}
```